c2rust-analyze/src/main.rs

```diff
@@ -272,12 +272,24 @@ fn is_build_script(at_args: &[String]) -> anyhow::Result<bool> {
     Ok(bin_crate_name().is_none() && is_bin_crate(at_args)?)
 }
 
+/// Check whether "no Cargo" mode is enabled.  In this mode, `c2rust-analyze` behaves like `rustc`
```

Comment:
Would it be useful to also have a command line flag, e.g., `c2rust-analyze --rustc-mode`?

Reply:
Currently we don't parse the command line arguments in rustc / rustc-wrapper mode - we just pass through whatever args we get to rustc's own argument handling.  All `c2rust-analyze`-specific configuration in these modes is done through env vars.  If we decide in the future to change this, then at that point I think it would make sense to have `--rustc-mode` (or a subcommand, maybe) as well.